tensortrade/oms/wallets/wallet.py

```diff
@@ -343,13 +343,14 @@ def transfer(source: 'Wallet',
 
         q = quantity.size
         c = commission.size
+        qc = (q + c) if c > 0.0 else q
```

Comment:
This looks to be a different change, nothing to do with Binance. If you insist on doing this, you need to tell us what else you are dragging into a commit and why.

Reply:
We simply cannot ignore this, because it is included in this PR. Either it gets fully merged, or it does not.

To fix this, someone has to push commits to revert these changes (as I have sometimes done in the past), and it will also change the code present in your own branch on which the PR is based (if you checked to allow this behavior which is the default).

You can surely understand that this is really difficult to intercept each time. The clean solution on your side is to create your branches starting from our master, not a local edited master in your fork.